addons/hr_timesheet/report/timesheets_analysis_report.py

```diff
@@ -24,6 +24,7 @@ class TimesheetsAnalysisReport(models.Model):
     amount = fields.Monetary("Amount", readonly=True)
     unit_amount = fields.Float("Hours Spent", readonly=True)
     partner_id = fields.Many2one('res.partner', string="Partner", readonly=True)
+    milestone_id = fields.Many2one('project.milestone', readonly=True)
```

Comment:
We should probably just do a related field as well, or we should do a left join on project task to get the milestones in the query. 🙂 

As you did in another task for another field. 🙂